README.md

```diff
@@ -10,6 +10,13 @@ This Gem enables users to develop robotic simulations through ROS2 tools and com
   * [ROS 2 Galactic](https://docs.ros.org/en/galactic/Installation.html) with Ubuntu 20.04.
   * [ROS 2 Humble](https://docs.ros.org/en/humble/Installation.html) with Ubuntu 22.04.
 
+#### Additional ros packages required
+
+Replace `<distro>` with ROS 2 distribution name (galactic, humble, ..).
+
+* Ackermann messages: `sudo apt install ros-<distro>-ackermann-msgs`
+* Control toolbox `sudo apt install ros-<distro>-control-toolbox`
+
```

Comment:
Maybe it would be more convenient for the user to have two separate sections for each distribution?

Also, ROS2 Gem probably requires some more packages to work, which might not be available in ROS2 installation (like `urdfdom`, `sensor_msgs` etc), then maybe it would be useful to add it all here? Unless of course, they are in the ROS2 Desktop installation. In this case, however, it should be clearly mentioned that the installation should be a Desktop one.

````suggestion
#### Additional ROS 2 packages required

##### Galactic

```
sudo apt install ros-galactic-ackermann-msgs ros-galactic-control-toolbox
```

##### Humble

```
sudo apt install ros-humble-ackermann-msgs ros-humble-control-toolbox
```

````

Reply:
I guess it is more convenient for the user to just copy and paste it, but also if we support even more distributions there is a lot of repetition here. As long as we do not need different steps for different..

I added a handy command and additional explaination